### DISCUSSION TITLE
nanoVNA-Saver herunterladen

### DISCUSSION THREAD

**Discussion by JuergenThierbach** [NanoVNA-Saver/nanovna-saver#692] [Q&A]
Wo finde ich für meinen nanoVNA-H, HW-Version 3.6_MS sowie Version 1.2.14 (aus Configuration) einen passende Saver-Download?
Wie lade ich den herunter und was ist beim Installieren zu beachten?
Bei GitHub gelange ich beim Suchen eines Downloads/Releases immer auf die angehängte Seite (Sorry, keine Binärdatei ... ). Was bedeutet dies[Mappe3.xlsx](https://github.com/NanoVNA-Saver/nanovna-saver/files/14283854/Mappe3.xlsx) und wie ist damit weiter zu verfahren?
Ich wäre glücklich, wenn ich das wüsste.   Jürgen

**Comment by Ho-Ro** [comment]
Click on **<> Code**
![image](https://github.com/NanoVNA-Saver/nanovna-saver/assets/12542359/1941af17-b647-4a73-b52d-d39125eea0db)
... then **Download ZIP**
![image](https://github.com/NanoVNA-Saver/nanovna-saver/assets/12542359/e0f12b49-7e1f-4414-a271-993d14e2af0b)
Unzip into a directory and start it with `python nanovna-saver.py` from a command window. Check the error messages for missing packages and install them with `pip install <MISSING_PACKAGE>` until it works. No need to use an `*.exe` file.

And what's the purpose of your `Mappe3.xlsx` file?

**Reply by JuergenThierbach** [reply]
Vielen Dank, Jürgen

**Reply by JuergenThierbach** [reply]
Was ich aber in der entpackten Zip-Datei vermisse, ist eine exe-Datei!
Oder anders gefragt, wie starte ich denn nun den NanoVNA-Saver?
Jürgen

**Reply by JuergenThierbach** [reply]
Nachtrag: ... Ich möchte gern mit Windows arbeiten.

**Reply by Ho-Ro** [reply]
As I wrote above:
Unzip into a directory and start it with `python nanovna-saver.py` from a command window from this directory.

**Reply by JuergenThierbach** [reply]
Danke für die Info.

1. Mein Problem ist, dass ich absolut keine Kenntnisse im Umgang mit Phyton habe!
Ohne eine ganz detaillierte Anleitung bin ich dabei leider hilflos. Kann mir jemand helfen?
2. Gibt es denn keine Windows-Lösung?
Jürgen

**Comment by zarath** [comment]
Binaries für Windows funktionieren ja wieder...
